doc/development/squid.md

```diff
@@ -71,182 +85,218 @@ Parameters:
 * web3Dto - Web3 client wrapped in a DTO object to avoid coupling with specific web3 specific libraries
 * providerDto - Ocean Provider client wrapped in a DTO object
 
-This method returns an instance of the **OceanManager** class. It allows to get access to the other Ocean core methods
+This method returns an instance of the **Ocean** class. It allows to get access to the other Ocean core methods
 
-## Ocean
 
-Interface with core Ocean functions
+In Python 
+Instantiate an ocean object
+
+
+```python
+   ocean = Ocean(keeper_url = '', provider_url = '', [web3=web3Object] ...)
+```
 
-### Balance
+## Ocean
 
+Interface with core Ocean functions
 
 #### Functions
 
+* **getAccounts** - Returns all avalible accounts loaded via a wallet, or by Web3.
+```
+array[address, tokenBalance, etherBalance]= ocean.getAccounts()
+```
+
 * **getOceanBalance** - SYNC. The only parameter required is an account address (ie. 0x6309b5dd9245278a7fdfb2186dfb80583caeadc7). Returns the Ocean Tokens balance for that account.
 ```
-oceanBalance= getOceanBalance(address)
+oceanBalance= ocean.getOceanBalance(address)
 ```
 
 * **getEtherBalance** - SYNC. The only parameter required is an account address. Returns the Ether balance for that account.
 ```
-ethBalance= getEtherBalance(address)
+ethBalance= ocean.getEtherBalance(address)
 ```
 
 * **getBalance** - SYNC. The only parameter required is an account address. Returns the Ocean tokens and Ether balance for that account.
 ```
-balance= getBalance(address)
+balance= ocean.getBalance(address)
 ```
 
-
-### Tokens
-
-#### Functions
-
-* **requestTokens** - SYNC. Request a number of Ocean Tokens. Returns a boolean to know if everything was right.
-
+* **requestTokens** - SYNC. Request a number of Ocean Tokens for an address. Returns a boolean to know if everything was right.
 ```
-result= requestTokens(amountTokens)
+result= ocean.requestTokens(account_address, amountTokens)
 ```
 
+* **searchAssets** - SYNC. Given a search query, returns a list of the DDO's matching with that query
+```
+array[assetDID]= ocean.searchAssets(searchQuery)
```

Comment:
I think the return object should be an array of Asset's (Metadata)

Reply:
I'm just being doing embedded stuff and I'm always trying to reduce memory usage. 
So yes we should be returning objects in the searches.